src/components/LandingPage/Categories/Categories.jsx

```diff
@@ -1,16 +1,39 @@
-import { Link } from "react-router-dom"
+import { Link, useNavigate } from "react-router-dom"
+import { useEffect, useState } from "react"
+import axios from "axios"
+import { useFilter } from "../../../contexts/FilterContext"
 import "./Categories.css"
-import { categories } from "../../../backend/db/categories"
 
 export default function Categories(){
+    const {filterDispatch} = useFilter()
+    const [categories, setCategories] = useState([])
+    const navigate = useNavigate()
+
+    useEffect(()=>{
+        const fetchCategories = async () => {
+            try{
+                const response = await axios.get('/api/categories')
+                setCategories(response.data.categories)
+            } catch(error) {
+                console.log(error)
```

Comment:
don't console log errors may be you can show that to user to make a good user experience